docs/themes/avocadocs/layouts/partials/scripts/global.html

```diff
@@ -3,3 +3,7 @@
 <script src="{{ "docsuikit/assets/vendor/jquery-migrate/dist/jquery-migrate.min.js" | relURL }}"></script>
 <!-- <script src="{{ "docsuikit/assets/vendor/popper.js/dist/umd/popper.min.js" | relURL }}"></script> -->
 <script src="{{ "docsuikit/assets/vendor/bootstrap/dist/js/bootstrap.min.js" | relURL }}"></script>
+{{ if (.Params.mermaid) }}
+<!-- MermaidJS support -->
+<script async src="https://unpkg.com/mermaid@9.0.0/dist/mermaid.min.js"></script>
```

Comment:
Depends what you mean by safe. In terms of functionality, it's safer to use a specific version rather than the latest or something that might have breaking changes. This is like having the specific version in `package-lock`.

It's only used on specific pages, yes.

Hugo is not really intuitive in this way, but I think I did manage to add it as a regular dependency. But honestly, this is not the most dangerous attack vector.